File: src/main/java/com/softwareplace/exception/IllegalConstraintsException.java
package com.softwareplace.exception;

import java.util.List;

public class IllegalConstraintsException extends Exception {

	private final List<String> errors;

	public IllegalConstraintsException(String message, List<String> errors) {
		super(message);
		this.errors = errors;
	}

	public List<String> getErrors() {
		return errors;
	}
}

File: src/main/java/com/softwareplace/exception/IllegalConstraintsException.kt
package com.softwareplace.exception;

import java.util.List;
import java.util.Map;

public class IllegalConstraintsException extends Exception {

	private final Map<String, List<String>> errors;

	public IllegalConstraintsException(String message, Map<String, List<String>> errors) {
		super(message);
		this.errors = errors;
	}

	public Map<String, List<String>> getErrors() {
		return errors;
	}
}
